src/lucky/params.cr

```diff
@@ -70,10 +107,43 @@ class Lucky::Params
     @_form_params ||= HTTP::Params.parse(body)
   end
 
+  @_multipart_params : Hash(String, String)?
+
+  private def multipart_params : Hash(String, String)
+    @_multipart_params ||= parse_multipart_request[0]
+  end
+
+  @_multipart_files : Hash(String, Tempfile)?
+
+  private def multipart_files : Hash(String, Tempfile)
+    @_multipart_files ||=  parse_multipart_request[1]
```

Comment:
You *might* be able to use `second`. I'm not sure if Crystal has that method, but worth a try

Reply:
Worth a try indeed.